Resources/Locale/en-US/clothing/clothing-speed.ftl

```diff
@@ -0,0 +1,9 @@
+# Clothing speed examine
+clothing-speed-examinable-verb-text = Clothing
+clothing-speed-examinable-verb-message = Examine the clothing speed values.
+clothing-speed-increase-equal-examine = This increases your speed by [color=yellow]{$walkSpeed}%[/color].
+clothing-speed-decrease-equal-examine = This decreases your speed by [color=yellow]{$walkSpeed}%[/color].
+clothing-speed-increase-run-examine = This increases your running spead by [color=yellow]{$runSpeed}%[/color].
+clothing-speed-decrease-run-examine = This decreases your running spead by [color=yellow]{$runSpeed}%[/color].
+clothing-speed-increase-walk-examine = This increases your walking spead by [color=yellow]{$walkSpeed}%[/color].
+clothing-speed-decrease-walk-examine = This decreases your walking spead by [color=yellow]{$walkSpeed}%[/color].
```

Comment:
```suggestion
clothing-speed-increase-run-examine = This increases your running speed by [color=yellow]{$runSpeed}%[/color].
clothing-speed-decrease-run-examine = This decreases your running speed by [color=yellow]{$runSpeed}%[/color].
clothing-speed-increase-walk-examine = This increases your walking speed by [color=yellow]{$walkSpeed}%[/color].
clothing-speed-decrease-walk-examine = This decreases your walking speed by [color=yellow]{$walkSpeed}%[/color].
```

Reply:
Though you can probably just pass in increases / decreases + <walking speed / running speed / speed> and reduce the loc.

Reply:
> Though you can probably just pass in increases / decreases + <walking speed / running speed / speed> and reduce the loc.

If we want to keep it able to be localized I think this, despite being clunky, is best